draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -1703,13 +1714,13 @@ the target elliptic curve.
 
 - MUST be built on primitives designed for use in applications requiring
 cryptographic randomness. As examples, a secure stream cipher is an appropriate
-primitive, whereas a Mersenne twister pseudorandom number generator is not.
+primitive, whereas a Mersenne twister pseudorandom number generator {{MT98}} is not.
 
-- MUST NOT use any form of rejection sampling.
+- MUST NOT use rejection sampling.
 
 - MUST give independent values for distinct (msg, DST, length) inputs.
-Meeting this requirement is slightly subtle.
-As a simplified example, hashing the concatenation msg || DST does not work,
+Meeting this requirement is subtle.
+As a simplified example, hashing msg || DST does not work,
```

Comment:
Does `||` need escaping as elsewhere?

Or maybe the escaping is unnecessary?

Reply:
It doesn't look like it (I see it render fine)?